Code/GraphMol/Substruct/SubstructMatch.cpp

```diff
@@ -35,6 +35,20 @@
 
 namespace RDKit {
 namespace detail {
+
+namespace {
+bool hasChiralLabel(const Atom *at) {
+  return at->getChiralTag() == Atom::CHI_TETRAHEDRAL_CW ||
+         at->getChiralTag() == Atom::CHI_TETRAHEDRAL_CCW;
+}
+
+//! Decide if an atom comes from a query that describes a mol fragment
```

Comment:
This is actually just testing whether or not the degree of the atom is <4 and it hasn't had the implicit valence value calculated.